src/transaction/types.ts

```diff
@@ -56,7 +57,8 @@ export type KeyPair = {
 }
 
 export type SignOptions = {
-  signAs: string
+  signAs: string,
+  allowHighFee?: boolean
```

Comment:
I think `docgen` needs to be run again to include this.

Reply:
Done